cernopendata_client/__init__.py

```diff
@@ -13,4 +13,4 @@
 
 from .version import __version__
 
-__all__ = ('__version__')
+__all__ = "__version__"
```

Comment:
`__all__ = ("__version__", )` <- should be a sequence of strings 📦

Reply:
Thanks for the catch :wink: Fix added to PR #28.